docs/revsets.md

```diff
@@ -15,8 +15,9 @@ The words "revisions" and "commits" are used interchangeably in this document.
 ## Hidden revisions
 
 Most revsets search only the [visible commits](glossary.md#visible-commits).
-Other commits are only included if you explicitly mention them (e.g. by commit
-ID, `<name>@<remote>` symbol, or `at_operation()` function).
+The visible commits are those reachable from `visible_heads()`. Other commits
```

Comment:
Should we avoid "reachable" here so the reader doesn't mistake it for the meaning in the `reachable()` revset function? We could say "The visible commits are those in `visible_heads()` and their ancestors.".

Reply:
Good idea